src/legacy/ui/public/visualize/loader/utils/query_geohash_bounds.ts

```diff
@@ -47,7 +49,9 @@ export async function queryGeohashBounds(vis: Vis, params: QueryGeohashBoundsPar
   });
 
   if (agg) {
-    const searchSource = vis.searchSource.createChild();
+    const searchSource = params.searchSource
```

Comment:
i need to check if i can remove this

Reply:
oh, no this needs to stay, instead of using searchsource from vis (which is no longer there) its using the one passed in.